star-sys/STAR/source/MmapArray.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MMAPARRAY_DEF
+#define MMAPARRAY_DEF
+
+#include "IncludeDefine.h"
```

Comment:
Seems to me this is only needlessly increasing compile times.  Better to directly include what you need.  Especially since `IncludeDefine.h` does all kinds of awful things like globally `using namespace std;`
```suggestion
#include <cstddef>
```